liquibase/changeLogs/capture/tables/fieldVisitReadings/fieldVisitReadings.sql

```diff
@@ -0,0 +1,24 @@
+create table if not exists ${AQTS_SCHEMA_NAME}.field_visit_readings
+(field_visit_readings_id                bigint generated by default as identity (start with 1)
+	,json_data_id                            bigint
+	,parameter                               text
+	,monitoring_method                       text
+	,field_visit_value                       jsonb
+	,unit                                    text
+	,uncertainty                             jsonb
+	,reading_type                            text
+	,manufacturer                            text
+	,model                                   text
+	,serial_number                           text
+	,field_visit_time                        timestamp
+	,comments                                text
+	,publish                                 text
+	,is_valid                                text
+	,reference_points_unique_id              text
+	,use_location_datum_as_reference         text
+	,reading_qualifier                       text
+	,reading_qualifiers                      jsonb
+	,ground_water_measurement                jsonb
+	,partition_number                        integer default extract( month from now() )
+)
+partition by list (partition_number);
```

Comment:
`value` and `time` from the sv json are likely keyword/reserved words in postgres, thanks for pointing that out @ayan-usgs .  They are not represented with `field_visit_` prefixes.